src/sparseml/integration_helper_functions.py

```diff
@@ -32,6 +33,27 @@ class Integrations(Enum):
     image_classification = "image-classification"
 
 
+def create_sample_inputs_outputs(
```

Comment:
we need the model here to actually create the outputs. right now I think you have it confused for labels

Reply:
Oh, you are right.